testdata/scripts/node/validate/valid.txtar

```diff
@@ -376,6 +376,7 @@ MaxBufferSize = 3
 SamplingInterval = '1s'
 MaxAllowedFinalityDepth = 10000
 FinalityTagBypass = true
+HeadTelemetryEnabled = false
```

Comment:
Does the prefix `Head` add value, given that we are already scoped to `HeadTracker`?
```suggestion
TelemetryEnabled = false
```

Why does this even need to be configurable? Why not have it enabled automatically, all of the time?

Reply:
done